addons/binding/org.openhab.binding.lgtvserial/README.md

```diff
@@ -0,0 +1,19 @@
+# LG TV control using serial port
```

Comment:
Would be lovely if you could spend 10 minutes and slightly extend the README.md, so that it roughly matches the template at https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/README.md.